provider/go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	github.com/hashicorp/aws-sdk-go-base/v2 v2.0.0-beta.32
 	github.com/hashicorp/terraform-provider-aws/shim v0.0.0
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/pulumi/pulumi-terraform-bridge/pf v0.15.0
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.56.0
+	github.com/pulumi/pulumi-terraform-bridge/pf v0.15.1
```

Comment:
let's use pf/v0.15.2 - it pulls in the bridge changes from 3.56, since I missed that we have to update both.

Reply:
Nice, sorry I missed that. Meant to use the latest.